__tests__/app.test.js

```diff
@@ -124,3 +123,56 @@ describe.only("get/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("post/api/articles/:article_id/comments", () => {
+  test("POST: 201 should post a comment object to corresponding article using id and respond with that object", () => {
+    const newComment = {
+      body: "This is the new comment on article 9",
+      author: "butter_bridge",
+    };
+    return request(app)
+      .post("/api/articles/9/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body[0]).hasOwnProperty("body");
+        expect(body[0]).hasOwnProperty("author");
+        expect(body[0]).hasOwnProperty("article_id");
```

Comment:
If we are expecting body to be an array would it be better to test everyone object in the array rather that the first?

Also when testing using objects there are some great matchers that may be of use to you here. Consider checking out objectContaining in the jest docs.

https://jestjs.io/docs/expect